environments/test.env

```diff
@@ -14,7 +14,7 @@ GOOGLE_CLIENT_ID="google-client-id"
 GOOGLE_CLIENT_SECRET="google-client-secret"
 GOOGLE_REDIRECT_URL="http://localhost:8000/v1/auth/google/callback"
 
-Max_Url_Count=10
-TokenExpiration=3600
+USER_MAX_URL_COUNT=10
```

Comment:
NIT: this could have been USER_MAX_URL_LIMIT

Reply:
We can take this later as USER_MAX_URL_COUNT is already added to the env and need to change in another PR also

Reply:
Then can you please create a separate issue for it and past the link here?

Reply:
https://github.com/Real-Dev-Squad/tiny-site-backend/issues/117